blog/nixos_remote_development/2023-11-16-Pilot.mdx

```diff
@@ -0,0 +1,199 @@
+---
+description: A minimal NixOS configuration to get started with remote development.
+slug: nixos-remote-development
+authors: shivaraj-bh
+tags: [nixos, minimal, flakes, nix]
+hide_table_of_contents: false
+---
+
+# NixOS Remote Development
+
+In this series of blog posts, we'll explore how to set up a remote development environment using NixOS. The goal of this post is to:
+- Create a minimal NixOS configuration.
+- Deploy the configuration (partition the disk and install the OS) to a remote machine over SSH (without a bootable USB).
+
+## Why?
+
+For starters, your system configuration is defined in code. This means that you don't have to login to see the state, like what users are present on the system, what services are running, etc. You can also deploy the configuration remotely.
+
+If you are someone who would like to use a powerful remote machine for development, while carrying around a lightweight laptop, then this post is for you.
+
+## The `flake.nix` file
+
+:::note
+If you are looking to just get started, you can skip this section and jump to the [next one](#deploying-the-configuration).
+:::
+
```

Comment:
Perhaps add a description of the flake as a paragraph here. Like `Here's a flake that contains a single NixOS configuration called "office"`